blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -128,33 +128,28 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
   def withoutAsynchronousChannelGroup: BlazeClientBuilder[F] =
     withAsynchronousChannelGroupOption(None)
 
-  def allocate(implicit F: ConcurrentEffect[F]): F[(Client[F], F[Unit])] =
-    tickWheelAllocate.flatMap {
-      case (scheduler, shutdownS) =>
-        connectionManager.map {
-          case (manager, shutdown) =>
-            (
-              BlazeClient.makeClient(
-                manager = manager,
-                responseHeaderTimeout = responseHeaderTimeout,
-                idleTimeout = idleTimeout,
-                requestTimeout = requestTimeout,
-                scheduler = scheduler,
-                ec = executionContext
-              ),
-              shutdown.guarantee(shutdownS)
-            )
-        }
-    }
+  def allocate(implicit F: ConcurrentEffect[F]): F[(Client[F], ExitCase[Throwable] => F[Unit])] =
```

Comment:
I think I saw discussion of this return type fly by on Gitter recently, but I can't find it now.  Do we want the `ExitCase` here or nah?

Reply:
Do we care about ExitCase ever?